### DISCUSSION TITLE
这个项目目前还在开发吗？如何参与？

### DISCUSSION THREAD

**Discussion by Astatine-213-Tian** [bangumi/frontend#636] [General]
有一定 React 开发经验，感兴趣参与开发。但是看好像好久没更新了？

**Comment by trim21** [comment]
噗，看错仓库名了。

因为目前没人写后端API（

**Reply by Astatine-213-Tian** [reply]
原来是这样（

**Comment by Ayase-252** [comment]
看看 server private（？忘名字了

**Reply by Astatine-213-Tian** [reply]
https://github.com/bangumi/server-private
这个现在都有什么需求？

**Reply by trim21** [reply]
> https://github.com/bangumi/server-private
> 这个现在都有什么需求？

有设计稿没api的部分都是需求（

**Reply by trim21** [reply]
我建议先把条目编辑的功能做完

**Reply by Astatine-213-Tian** [reply]
设计文档是指[这个](https://next.bgm.tv/p1/#/)吗？（我看这上面的好像都实现了）

**Reply by trim21** [reply]
看本仓库的wiki
